baseline/client/verbal-learning/verbal-learning.js

```diff
@@ -39,50 +40,106 @@ import remember_a_cue_animal_html from "./frag/remember_a_cue_animal.html";
 import remember_a_long_html from "./frag/remember_a_long.html";
 
 export class VerbalLearning {
+    constructor(setNum, segmentNum, getLastSegmentEndTime) {
```

Comment:
`getLastSegmentEndTime` should be a callback that returns the time of the last segment's completion as milliseconds since epoch (like `Date.now()`). It will only be called once because the first call's return value will be cached.